core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/util/CamelComponentUtils.java

```diff
@@ -280,6 +280,13 @@ protected static String getComponentJSon(String scheme, IProject project) {
         
         return json;
     }
+	
+	private static boolean isJarFile(File f) {
+    	if (f.isFile() && f.getName().toLowerCase().endsWith(".jar")) {
```

Comment:
I think you can directly return f.isFile() && f.getName().toLowerCase().endsWith(".jar")